app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class PhysicalServerDecorator < MiqDecorator
+  def fileicon
+    "svg/vendor-#{vendor.try(:downcase)}.svg"
```

Comment:
This is a bit scary:
- the idea behind decorators is to avoid using things like `vendor.downcase` and have explicit images here
- what if the `downcase` method isn't defined, `fileicon` will return with `'svg/vendor-.svg'` :scream: